### ISSUE TITLE
Push force

### ISSUE THREAD

**Issue by qrevel** [chartmuseum/helm-push#5]
Currently if the chart already exists, the `push` return a 409 error code:

```
/ # helm push yolo chartmuseum
Pushing yolo-0.1.0.tgz to chartmuseum...
Error: 409: yolo-0.1.0.tgz already exists
Error: plugin "push" exited with error
```

It would be nice to have a `-f, --force` option to overwrite the existing chart.

**Comment by jdolitsky** [comment]
@qrevel the overwrite is controlled on the server side with the `--allow-overwrite` / `ALLOW_OVERWRITE=true` option. I do like this idea though. It would take additional work server side to support the force option

**Comment by qrevel** [comment]
Thanks for the information.

Let's keep the issue open if you/someone find a way to implement `--force` on the client side.

**Comment by jdolitsky** [comment]
@qrevel Thanks again for this. This is now supported on master, as well as the "latest" chartmuseum docker image.

Please try it out! You can clone the repo and build/install from source:
```
git clone git@github.com:chartmuseum/helm-push.git
cd helm-push/
make remove; make build_mac install
```

Then just pass the `--force`/`-f` flag:
```
helm push --force mychart-0.1.0.tgz chartmuseum
```

Let me know if things work as expected.

**Comment by qrevel** [comment]
Sounds good, is it possible to get a plugin release so I can use it with `helm plugin install`?

Thanks

**Comment by jdolitsky** [comment]
@qrevel yes, just released v0.5.0, please git it a try. Make sure using "latest" chartmuseum image, no release there yet

**Comment by jdolitsky** [comment]
@qrevel also, you should be able to remove bash from your image now (#4)

**Comment by jdolitsky** [comment]
@qrevel were you able to try this?

**Comment by agolomoodysaada** [comment]
This seems to not work on `0.7.1`. Not sure if a regression was introduced.

```sh
$ helm plugin list
NAME	VERSION	DESCRIPTION
push	0.7.1  	Push chart package to ChartMuseum
```

```sh
$ helm push --force agolo-service chartmuseum
Pushing agolo-service-0.1.2.tgz to chartmuseum...
Error: 409: agolo-service-0.1.2.tgz already exists
Error: plugin "push" exited with error
```

**Comment by agolomoodysaada** [comment]
That worked! Thank you
